aesara/sparse/basic.py

```diff
@@ -355,8 +360,7 @@ def __getitem__(self, args):
         return ret
 
 
-class SparseVariable(_sparse_py_operators, Variable):
-    dtype = property(lambda self: self.type.dtype)
+class SparseVariable(_sparse_py_operators, TensorVariable):
```

Comment:
Why are the supertypes in a different order compared to the `SparseConstant` below?

Reply:
Not sure.